SalesforceFoundation/EDA
# Feature/xinnanli/naming empty string

# Critical Changes

# Changes
Fixes household naming bug when all the data in a contact is empty. Use comma instead of "and" to connect names from different contacts
# Issues Closed
Fixes #553
# New Metadata

# Deleted Metadata

# Testing Notes
(In addition to other household naming regression testing)
1. Go to HEDA setting page
2. Enable automatic household naming. Choose household as default account model, "Other" as Household Account Name Format. Fill in the other format with: {!Salutation} {!FirstName} Family.
3. Create a contact: Salutation = "Mr.", FirstName = "TestFN", LastName = "TestLN"
4. Make sure the household account name is Mr. TestFN Family
5. Create another contact under the same household account: Salutation = "", FirstName = "", LastName = "TestLN2"
6. Make sure the household account name is still Mr. TestFN Family
7. Update the second contact to: Salutation = "Ms.", FirstName = "TestFN2", LastName = "TestLN2"
8. Make sure the household account name is now Mr. TestFN and Ms. TestFN2 Family
9. Create another contact under the same household account: Salutation = "Mrs", FirstName = "TestFN3", LastName = "TestLN3"
10. Make sure the household account name is now Mr. TestFN, Ms. TestFN2 and Mrs. TestFN3 Family
11. Make sure other household account naming testing works too
